dkim/verify.go

```diff
@@ -86,11 +86,20 @@ type signature struct {
 	v string
 }
 
+// LookupTXT allows you customize the domain lookup to config timeout, dns resolver etc
```

Comment:
Doc comments should start with the name of the documented type. What about moving this comment above the `LookupTXT` field, and adding this doc comment for the struct: "VerifierOptions allows to customize the default signature verification behavior"?

Also I'd like to document what `LookupTXT` does and what's the default: "LookupTXT returns the DNS TXT records for the given domain name. If nil, net.LookupTXT is used."